### ISSUE TITLE
Astrometric microlensing

### ISSUE THREAD

**Issue by rpoleski** [rpoleski/MulensModel#53]
It's time to add astrometric microlensing to MM. Recently @skterry approached me and asked for it. We should start with **use cases - can somebody draft them?** 

We should think what are possible data conventions. I see a few possibilities:
- AO data - calibrated to (RA, Dec),
- Gaia - very accurate measurement in one direction and this direction changes from epoch to epoch,
- Gravity wide or plus - I'm guessing that soon we will have multiple epochs from that instrument, the deconstructed images have directions N/E, but are not calibrated to (RA, Dec) because there are no other stars in the field of view; also the microimages can be resolved or not.

We don't have to write use cases that take care of all above issues, but we have to remember one day we will have all these cases and use cases should leave possibility to incorporate these different kinds of information in future.

**Comment by skterry** [comment]
Hi @rpoleski, we are very happy to see this! If you would like any help with drafting the use cases, feel free to ask myself, @jluastro, or @caseylam.

**Comment by rpoleski** [comment]
Yes, I'd love to see use cases. @skterry, @jluastro, and @caseylam - you can start these. For reference, [examples/use_cases/](https://github.com/rpoleski/MulensModel/tree/master/examples/use_cases) is directory with our use cases. Some of still probably don't work, but in general you can see how we write them. The goal is to drive API of MM.
